app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.tsx

```diff
@@ -301,6 +286,13 @@ export const DirectoryLiveTable: React.FC<DirectoryTableProps> = (props) => (
               >
                 Results
               </Link>
+            ) : experiment.monitoringDashboardUrl ? (
```

Comment:
Oh I see, yeah I wouldn't do it as either results OR monitoring, while an experiment is live both could be available, this should just be a second check so if both results and monitoring are available both links appear 🙏

Reply:
Good point, @jaredlockhart! I wasn't sure if that condition was possible. Fixed in https://github.com/mozilla/experimenter/pull/6226/commits/f8393dbccf87555106eda13de92b3face880caf8.